docs/getting-started.md

```diff
@@ -164,26 +164,26 @@ In your **second shell** prompt:
 
   ```bash
     arch=`uname -m`
-    kernel_path="hello-vmlinux.bin"
+    kernel_path="$PWD/hello-vmlinux.bin"
 
     if [ ${arch} = "x86_64" ]; then
       curl --unix-socket /tmp/firecracker.socket -i \
           -X PUT 'http://localhost/boot-source'   \
           -H 'Accept: application/json'           \
           -H 'Content-Type: application/json'     \
-          -d '{
-              "kernel_image_path": \"${kernel_path}\",
-              "boot_args": "console=ttyS0 reboot=k panic=1 pci=off"
-          }'
+          -d "{
```

Comment:
P.S.: Replacing `'` with `"` is already done in another PR that we are going to merge soon: https://github.com/firecracker-microvm/firecracker/pull/1162